src/style.css

```diff
@@ -1,3 +1,70 @@
+/* stylelint-disable no-descending-specificity */
+/* stylelint-disable color-hex-case */
```

Comment:
these errors occur because you write colors in the uppercase, but you need to write them in the lowercase